src/stack/index.ts

```diff
@@ -144,10 +144,18 @@ class Stack {
    * @param {Object} params Optional parameters for the GET call
    * @return {Object} A promise object which will be resolved with details of the content type.
    */
-  getContentTypes(query = {}, params: { [key: string]: any } = {}): Promise<{ [key: string]: any }> {
+  getContentTypes(branch?: string, query = {}, params: { [key: string]: any } = {}): Promise<{ [key: string]: any }> {
```

Comment:
Why not just accept the branch parameter in params? So there won't be any change in signature but works fine when user started passing branch.
```
const { branch, ... optionParams } = params;
```

Reply:
done